lang/python/python3-django-etesync-journal/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=django-etesync-journal
+PKG_VERSION:=1.0.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=django-etesync-journal
+PKG_HASH:=716df1fc8670c79750e8a1e736b69ddbdbd238237565874ea9648387cc50626e
+
+PKG_LICENSE:=AGPL-3.0-only
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-django-etesync-journal
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=The server side implementation of the EteSync protocol.
+  URL:=https://www.etesync.com/
+  DEPENDS:=python3-django +python3-django-restframework +python3-light
+  VARIANT:=python3
+  MDEPENDS:=python3-django
+endef
+
+define Package/python3-django-etesync-journal/description
+  The reusable django app that implements the server side of the EteSync protocol.
+endef
+
+define Py3Package/python3-django-etesync-journal/filespec
++|$(PYTHON3_PKG_DIR)
+-|$(PYTHON3_PKG_DIR)/tests
```

Comment:
I think this needs the package directory name as well, e.g. `$(PYTHON3_PKG_DIR)/journal/tests` (replace `journal` with the right directory name).

`PYTHON3_PKG_DIR` resolves to `/usr/lib/python3.8/site-packages`.

The example in the readme is wrong (sorry) - will fix it soon.

Reply:
Hmm, I am not sure but I think `$(PYTHON3_PKG_DIR)/tests` is right. There are the following folders in `ipkg-install/usr/lib/python3.8/site-packages`:
`django_etesync_journal-1.0.2-py3.8.egg-info`
`journal`
`tests`
In `ipkg-mipsel_74kc/usr/lib/python3.8/site-packages` there are only the first two, the `tests` folder is not there.

Could remove the tests by patching the `setup.py`, too. Would that be better?

Reply:
@peter-stadler, could you please open a ticket and I'll fix this stupidity? Sorry about that.
Any idea what's the correct way of dealing with this? Should I have named `tests/` differently? Or should I just add an exclusion to `setup.py`, if so, how?

Reply:
@peter-stadler Oh I see (I didn't actually do a compile test :sweat_smile:) - for now I think it's okay to keep the filespec as-is. Depending on how this will be resolved upstream (usually packages ship their tests or examples inside of their package subdirectory), this may need to be updated later.

Reply:
I released 1.0.3 which fixes the shipping of tests so you don't need to patch setup.py.

Reply:
Thank you, I changed it to use 1.0.3 and removed the filespec :-)

Reply:
@tasn if you plan to rename it, it is at pypi with the name django-etesync-journal, maybe both names should be the same. But, I don't have a problem with different names and the name “journal” either.

Reply:
it's quite common for django packages to have one name for the pypi package and one name for the module. So e.g. the name of the pypi package would be `django-etesync-journal` and the module itself would be `etesync_journal`. Yeah, it's crazy, but that's how it is. I much prefer npm in that sense.

Another potential name for the module could be `etesync_backend` or something like that.

Anyhow, I thought about it, and it shouldn't be a blocker for this, as I can just rename it later on, and just adjust the server repo accordingly.

Reply:
I see, I am good with all these names.